code/development-env/config/local/image_config.json

```diff
@@ -0,0 +1,97 @@
+{
+  "images": {
+    "RSHINY": {
+      "displayName": "Shiny",
+      "type": "site",
+      "userSelectableVersion": true,
+      "versions": [
+        {
+          "default": true,
+          "displayName": "4.0.1",
+          "image": "nerc/rshiny:4.0.1"
+        },
+        {
+          "displayName": "3.5.3",
+          "image": "nerc/rshiny:3.5.3"
+        }
+      ]
+    },
+    "RSTUDIO": {
+      "displayName": "RStudio",
+      "type": "notebook",
+      "userSelectableVersion": true,
+      "versions": [
+        {
+          "default": true,
+          "displayName": "4.0.1",
+          "image": "nerc/rstudio:4.0.1",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        },
+        {
+          "default": false,
+          "displayName": "3.5.3",
+          "image": "nerc/rstudio:3.5.3",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        }
+      ]
+    },
+    "JUPYTERLAB": {
+      "displayName": "JupyterLab",
+      "type": "notebook",
+      "versions": [
+        {
+          "displayName": "f200ab964cea-dask-2.30.0",
+          "image": "nerc/jupyterlab:f200ab964cea-dask-2.30.0"
+        }
+      ]
+    },
+    "JUPYTER": {
+      "displayName": "Jupyter",
+      "type": "notebook",
+      "versions": [
+        {
+          "displayName": "f200ab964cea-dask-2.30.0",
+          "image": "nerc/jupyterlab:f200ab964cea-dask-2.30.0"
+        }
+      ]
+    },
+    "ZEPPELIN": {
+      "displayName": "Zeppelin",
+      "type": "notebook",
+      "versions": [
+        {
+          "displayName": "0.7.2.7",
+          "image": "nerc/zeppelin:0.7.2.7",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        }
+      ]
+    },
+    "NBVIEWER": {
+      "displayName": "NBViewer",
+      "type": "site",
+      "versions": [
+        {
+          "displayName": "latest",
+          "image": "jupyter/nbviewer:latest"
+        }
+      ]
+    },
+    "SPARK": {
```

Comment:
I've added an image type of `infrastructure` for images like Spark and Minio. Don't know if it is something worthwhile having? Happy to hear if it is not!